superset/models.py

```diff
@@ -1626,6 +1630,11 @@ def num_cols(self):
     def name(self):
         return self.datasource_name
 
+    @property
+    def schema_perm(self):
+        """Is not used. Created for consistency with SqlaTable."""
```

Comment:
I've seen people using dot notation in their druid datasource names, maybe we could try to split on `.` and take the first part of the string? Not a real requirement though.